src/go/rdctl/cmd/version.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+  "fmt"
+  "github.com/spf13/cobra"
+)
+
+// showVersionCmd represents the showVersion command
+var showVersionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Shows the CLI version.",
+	Long: `Shows the CLI version.`,
+  RunE: func(cmd *cobra.Command, args []string) error {
+    _, err := fmt.Printf("rdctl client version: %s, server version: %s\n", clientVersion, apiVersion)
```

Comment:
```suggestion
    _, err := fmt.Printf("rdctl client version: %s\nTarggeting server version: %s\n", clientVersion, apiVersion)
```
Unlike kubectl, we're not prodding the server to make it return its version. Speaking of, we don't have an endpoint on the server to report the API version it understands… that can be later.

Reply:
It's easier to process this output when it's all on a single line, but fine on adding "targeting".